landoui/templates/partials/layout.html

```diff
@@ -13,7 +13,7 @@
 {% include "partials/navbar.html" %}
 <section class="hero is-light is-bold">
   <div class="container">
-    <p class="subtitle">{% block banner_text %}{% endblock %}</p>
+    <div class="subtitle"></div>
```

Comment:
To clarify, if you remove this line in the layout, then you have to remove it from the revision.html page as you have already done in the signout page :)